src/core_codemods/remove_debug_breakpoint.py

```diff
@@ -0,0 +1,29 @@
+import libcst as cst
+from typing import Union
+from codemodder.codemods.api import BaseCodemod, ReviewGuidance
+from codemodder.codemods.utils_mixin import NameResolutionMixin, AncestorPatternsMixin
+
+
+class RemoveDebugBreakpoint(BaseCodemod, NameResolutionMixin, AncestorPatternsMixin):
+    NAME = "remove-debug-breakpoint"
+    SUMMARY = "Remove Breakpoint"
+    REVIEW_GUIDANCE = ReviewGuidance.MERGE_WITHOUT_REVIEW
+    DESCRIPTION = "Remove calls to builtin `breakpoint` or `pdb.set_trace."
+    REFERENCES: list = []
+
+    def leave_Expr(
+        self, original_node: cst.Expr, _
+    ) -> Union[cst.Expr, cst.RemovalSentinel]:
+        match call_node := original_node.value:
+            case cst.Call():
+                if self.find_base_name(
+                    call_node
+                ) == "breakpoint" and self.is_builtin_function(call_node):
```

Comment:
Would it be useful if I changed `find_base_name` to prepend `builtin.` to builtin functions?

Reply:
yeah it could clean up multiple checks where we also do `and self.is_builtin_function(call_node):`